### PR TITLE
Releases

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Added object representation for GitHub releases.
- What about pagure? Is there anything that can be used?

### CODE REVIEW

**Comment by TomasTomecek** [comment]
>     * What about pagure? Is there anything that can be used?

https://pagure.io/freshmaker/releases

File: ogr/abstract.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import datetime
 from enum import Enum
 from typing import Optional, Match, List, Dict
+from urllib.request import urlopen
```
Comment (TomasTomecek) [inline ogr/abstract.py]: I would ask that why not requests, but the code seems fairly easy

File: ogr/abstract.py [hunk 2]
```diff
@@ -48,6 +49,32 @@ def __init__(
...
+    def __init__(
+        self,
+        title: str,
+        body: str,
+        tag_name: str,
+        url: str,
+        created_at: str,
+        tarball_url: str,
+    ) -> None:
+        self.title = title
```
Comment (TomasTomecek) [inline ogr/abstract.py]: please provide comments on what these are and represent, ~~ideally with examples~~, right, it's abstract

File: ogr/services/github.py [hunk 3]
```diff
@@ -140,6 +141,17 @@ def _prcomment_from_github_object(raw_comment: GithubIssueComment) -> PRComment:
...
+    @staticmethod
+    def _release_from_github_object(raw_release: GithubRelease) -> Release:
+        return Release(
+            title=raw_release.title,
+            body=raw_release.body,
+            tag_name=raw_release.tag_name,
+            url=raw_release.url,
+            created_at=raw_release.created_at,
+            tarball_url=raw_release.tarball_url,
+        )
```
Comment (TomasTomecek) [inline ogr/services/github.py]: Ha! But here the examples would be lovely.

File: ogr/abstract.py (hunk 1 above)
Comment (lachmanfrantisek) [reply ogr/abstract.py]: ...PyCharm auto imports...

**Comment by lachmanfrantisek** [comment]
> > ```
> > * What about pagure? Is there anything that can be used?
> > ```
> 
> 
> https://pagure.io/freshmaker/releases

I need to look on the Pagure API...

**Comment by rpitonak** [comment]
@lachmanfrantisek what is the state of this PR now? I would like to look at `packit status`

**Comment by lachmanfrantisek** [comment]
> @lachmanfrantisek what is the state of this PR now? I would like to look at packit status

It's still WIP. We can close this one and open the new one from this repo, so you can continue to work on it. Is that OK?

**Comment by rpitonak** [comment]
@lachmanfrantisek sure we can do it like that.

**Comment by lachmanfrantisek** [comment]
Closing in favor of #32

**Pull request closed without merging**